.github/actions/to-api-contract-tests/entrypoint.sh

```diff
@@ -122,4 +122,4 @@ truncate --size=0 traffic.ops.log # Removes output from previous API versions an
 
 cd "../testing/api_contract/v$INPUT_VERSION"
 
-pytest -rA --to-user admin --to-password twelve12 --to-url https://localhost:6443/api/4.0
+pytest -rA --to-user admin --to-password twelve12 --to-url https://localhost:6443/api/4.1
```

Comment:
4.1 is not stable/released. You aren't writing tests for APIv5, so I'm guessing you don't want to be writing them for unstable APIs?

Reply:
Sure, I will remove this test case as this endpoint is not a part of APIv4.